applications/osb-portal/deploy/nginx.conf

```diff
@@ -0,0 +1,82 @@
+
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    gzip  on;
+
+    server {
+        listen       80;
+        server_name  localhost;
+
+        #charset koi8-r;
+        #access_log  /var/log/nginx/host.access.log  main;
+
+        location / {
+            root   /usr/share/nginx/html;
+            index  index.html index.htm;
+        }
+
+        #
+        location ~ ^/api/(?<subdomain>.*?)/(?<path>.*$) {
+            resolver kube-dns.kube-system.svc.cluster.local;
+            set $bcknd http://$subdomain.__REALM__.svc.cluster.local:8080/$path;
```

Comment:
Like it, looks very generalizable, only improvement I may suggest is using helm variables rather than sed substitutions